assets/i18n/en.json

```diff
@@ -445,7 +445,7 @@
     "I18N_SPLASH_SUBTITLE": "Oppia makes it easy to create interactive lessons that educate and engage.",
     "I18N_SPLASH_THIRD_EXPLORATION_DESCRIPTION": "Oppia lets you <a href=\"/creator_dashboard?mode=create\">create</a> and <a href=\"/library\">share</a> explorations on a wide range of subjects, limited only by your imagination.",
     "I18N_SPLASH_TITLE": "Think outside the books.",
-    "I18N_START_HERE": "Start Here!",
+    "I18N_START_HERE": "Click here to start",
```

Comment:
Hi @muxout -- please note that these changes need to be tested manually/visually too! Here's how it currently looks:

![starting](https://user-images.githubusercontent.com/10575562/28992672-b4b27582-7957-11e7-8f59-948c8947ac05.png)

Would it be possible to make the font smaller, and maybe preserve the exclamation mark at the end? I don't know if the arrow can be programmatically generated but that might be a good idea too.